### ISSUE TITLE
Date selection widget for Decisions Detail tab makes it difficult to enter dates in the past

### ISSUE THREAD

**Issue by leilatite** [agilepro/cognoscenti#623]
I'm not able to edit the year in the widget, so to get a date in the year 2000 I have to click back one month at a time. This makes the [saving the wrong year](https://github.com/agilepro/cognoscenti/issues/621) issue more frustrating because I have to do all those clicks all over again to update the decision record with the correct year.

**Comment by agilepro** [comment]
Yes.   We have gone through a couple.  I was trying to find something that is the right balance between number of clicks and ease of picking the normal cases.  

I also believe that the sliders for time is a bad idea.  It is much easier to just type the time.  Discovered this on my Apple phone where there are sliders to set the alarm time -- and then, for a little while I could just type the time, and soooo much easier.  This widget should be improved and there are many available to move to, just need to the time to integrate it.
